### PR TITLE
backdrop mask bug fix

### PR DESCRIPTION
**Pull request by zhuxingwei**

### CODE REVIEW

File: Runtime/ui/renderer/cmdbufferCanvas/rendering/canvas_impl.cs [hunk 1]
```diff
@@ -127,30 +125,32 @@ public partial class PictureFlusher {
             this._currentLayer = layer;
 
             if (paint.backdrop != null) {
-                if (paint.backdrop is _BlurImageFilter) {
-                    var filter = (_BlurImageFilter) paint.backdrop;
+                var points = new uiOffset[4];
```
Comment (yczhangsjtu) [inline Runtime/ui/renderer/cmdbufferCanvas/rendering/canvas_impl.cs]: Will this cause GC?

Comment (zhuxingwei) [reply Runtime/ui/renderer/cmdbufferCanvas/rendering/canvas_impl.cs]: uiOffset is struct, it should be gc free ?

Comment (yczhangsjtu) [reply Runtime/ui/renderer/cmdbufferCanvas/rendering/canvas_impl.cs]: I'm not sure about an array of structs.

Comment (zhuxingwei) [reply Runtime/ui/renderer/cmdbufferCanvas/rendering/canvas_impl.cs]: has tested a bit and turns out that array of structs is not gc free. try to fix this in the newer commit and please review the changes. Thanks a lot :-)

**Pull request merged**